test/src/041-rocache/main

```diff
@@ -1,37 +1,74 @@
 
 cvmfs_test_name="Read-only cache"
 
+lazy_umount() {
+  local dir=$1
+  if running_on_osx; then
+    sudo umount "$dir"
```

Comment:
Ah, this function isn't used at all on OS X, right?  Then you can just remove the OS X branch in this function.